pages/index.js

```diff
@@ -2,7 +2,34 @@ import Head from 'next/head'
 import Image from 'next/image'
 import styles from '../styles/Home.module.scss'
 
+
+
 export default function Home() {
+  return (
+    <div className={styles.body}>
```

Comment:
tady jsem fixnul ten error, proč se to stalo nas ted nemusi vubec zajimat. Jde jen o to, ze tady ten prvni div bude prvni uvnitr body a tys tu mel primo body. O to html, head a body se stara next.js.

Reply:
Aha, takže html, head a body nemusím vůbec zmiňovat? Ještě koukám, že jsi u toho <div> vymazal className, má to nějaký důvod? Jestli jo, tak teď nevím, jak např. pro celý <div> nastavit jednotné písmo. K tomu mám další otázku, v scss vlastně můžu upravovat jen to, co má nastavený className v jsx?

Reply:
ano html head body nezminujem, protoze pouzivame tenhle framework. Ten nam ulehcuje tady tu praci, co se porad opakuje. 

Vymazal jsem className, co byla na body a presunul jsem to css do souboru `globals.css` nebo tak nak. Udelal jsem to, protoze zase ten framework me k tomu nuti. v tom souboru uz byli nejaky veci pro body, tak neni dobre to mit na dvou mistech.